openquake/hmtk/tests/seismicity/catalogue_test.py

```diff
@@ -82,6 +87,15 @@ def setUp(self):
             [1960, 5.0],
         ])
 
+    def write_catalogue(self):
+        # Test the export of a catalogue in csv format
+        inp_file = os.path.join(CURRENT_DIR, 'data/test_write_input.csv')
+        out_file = os.path.join(CURRENT_DIR, 'data/test_write_output.csv')
```

Comment:
This is not point in committing the output file, since it is generated. Also, in the test the output file should be written on a temporary location (write permissions on the source directory cannot be assumed) and removed at the end of the test, after the comparison.